wordpress-mobile/WordPress-iOS
# Onboarding Improvements: Reuse Post Sign-up Interstitial if there's no sites

Ref: https://github.com/wordpress-mobile/WordPress-iOS/pull/17520#issuecomment-976733832

## Description

This PR simplifies the Login flow for when there's no sites yet by reusing the Post Sign-up Interstitial screen.

Login logic for when there's no site yet:
- Before: Post Post Sign-up Interstitial screen shown on first login, and no sites Epilogue variation shown on subsequent logins
- After: Post Sign-up Interstitial screen shown on every login

Before | After
-- | --
<img src="https://user-images.githubusercontent.com/6711616/142911540-eafa6c2f-7d4f-4732-9cf3-5c775b4bcea5.png" width=200> | <img src="https://user-images.githubusercontent.com/6711616/142620725-b2153bdd-0c3b-44d8-8744-f7ac1ebea5f7.PNG" width=200> 

## How to test
1. Log out / log in to a site w no sites
2. ✅ Make sure you see the Post Signup Interstitial screen
3. Repeat steps 1,2
4. ✅ Make sure you see the Post Signup Interstitial screen again

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
